website/project/views/contributor.py

```diff
@@ -378,6 +378,55 @@ def project_manage_contributors(auth, node, **kwargs):
     return {}
 
 
+@no_auto_transaction
+@must_be_valid_project  # injects project
+@must_have_permission(ADMIN)
+@must_not_be_registration
+def project_remove_contributor(auth, **kwargs):
```

Comment:
(I guess we already [established](https://github.com/CenterForOpenScience/osf.io/pull/4799/files#r49227813) that `project_remove_contributor` is copied from `project_manage_contributors`).